### PR TITLE
Proposal to bump WETH supply cap [cUSDCv3]

### PR DESCRIPTION
**Pull request by jflatow**
Some risk parameter tweaks for cUSDCv3.

TODO: 
 - [x] Forum post (Gauntlet?)
 - [x] Update proposal text

### CODE REVIEW

**Review by scott-silver — APPROVED** [review]
🙌

File: deployments/mainnet/usdc/migrations/1669668117_raise_weth_supply_cap.ts [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+    const event = txn.events.find(event => event.event === 'ProposalCreated');
+    const [proposalId] = event.args;
+
+    trace(`Created proposal ${proposalId}.`);
+  },
+
+  async verify(deploymentManager: DeploymentManager) {
+    const {
+      comet,
+      COMP,
```
Comment (kevincheng96) [inline deployments/mainnet/usdc/migrations/1669668117_raise_weth_supply_cap.ts:53]: No longer needed

File: deployments/mainnet/usdc/migrations/1669668117_raise_weth_supply_cap.ts [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+      WETH,
+    } = await deploymentManager.getContracts();
+
+    const compInfo = await comet.getAssetInfoByAddress(COMP.address);
```
Comment (kevincheng96) [inline deployments/mainnet/usdc/migrations/1669668117_raise_weth_supply_cap.ts:57]: Same here

**Review by kevincheng96 — APPROVED** [review]
✅

File: deployments/mainnet/usdc/migrations/1669668117_raise_weth_supply_cap.ts (hunk 1 above)
Comment (jflatow) [reply deployments/mainnet/usdc/migrations/1669668117_raise_weth_supply_cap.ts:53]: True 🎣

**Pull request merged**